### ISSUE TITLE
Provides a --help to show quick documentation about all available command-line options

### ISSUE THREAD

**Issue by luzfcb** [ValveSoftware/steam-for-linux#7849]
#### Your system information

* Steam client version (build number or date): Jun 8, 2021 at 22:23:41
* Distribution (e.g. Ubuntu): Ubuntu 20.04
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:

Recently a Steam update caused Steam to completely stop opening correctly and it didn't show me any error messages.
In my case, after a lot of google search, I found that I could run[ `STEAM_RUNTIME=0 steam` which showed me that I needed `libpipewire-0.3:i386`](https://github.com/ValveSoftware/steam-for-linux/issues/7847#issuecomment-857765461) (package that there is no i386 version published in Ubuntu repositories), and [this comment ](https://github.com/ValveSoftware/steam-for-linux/issues/7847#issuecomment-857668644) let me know of the existence of a `--reset `command line modifier on steam. The `--reset` solves the problem in my case.

Having a --help with the --reset documentation would have saved me a lot of work and time spent searching google for a solution.

I would expect a --help command to be made available containing at least the commands that can be publicly released and that can be useful for people to solve their own problems without having to consult outside support.

#### Steps for reproducing this issue:

1. Open the terminal
2. Running `steam --help`
3. The steam is opening UI instead of showing the documentation as we expect in a Linux software.

**Comment by smcv** [comment]
> I found that I could run `STEAM_RUNTIME=0 steam`

For the record: this is unsupported and is not expected to work. If it reports that libraries are missing, that doesn't necessarily mean you need those libraries when running normally.

You *can* run this, but there are lots of things you can run that don't work.

**Comment by memory-thrasher** [comment]
I agree. Some time ago I learned of `--console` which adds a debug tab to steam. Also, when diagnosing an unrelated issue, I discovered that the steam start scripts check the environment for a `DEBUGGER` variable, which it then uses to launch steam (ie `DEBUGGER=gdb steam`) without the launcher scripts cluttering things up. All took effort to find, effort that could have been spent dealing with the actual problem.

Bottom line, if valve does not want users to know about debugging measures, they should make steam actually work, so it does not need debugging. If you're going to put out buggy software, and then complain that reporting users don't put enough effort into diagnosing the problem before reporting it, you should make the debugging functionality more accessible.
